spec.md

```diff
@@ -639,7 +639,7 @@ name:
 ```php
 public function process(string $algorithm, ...$parts)
 {
-    // processing
+    // Method/function body
```

Comment:
This is definitely a method and not a function since you can't use visibility modifiers with functions

Reply:
Agreed. Laziness on my part.

Fixed.